scripts/monitor-performance.sh

```
@@ -0,0 +1,77 @@
+echo "=============================================="
+echo ""
+
+# Check active environment
+ACTIVE_ENV=$(grep ACTIVE_COLOR /home/cvat/spheroseg-app/.active-environment | cut -d'=' -f2)
+echo "📍 Active Environment: $ACTIVE_ENV"
+echo ""
+
+# Function to check container health
+check_health() {
+    local container=$1
+    local health=$(docker inspect --format='{{.State.Health.Status}}' $container 2>/dev/null || echo "not found")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)